torch/csrc/jit/frontend/schema_matching.h

```diff
@@ -18,6 +18,10 @@ struct MatchedSchema {
   c10::OptNameList return_field_names;
 };
 
+TORCH_API int calculateNecessaryArgs(
+    const std::vector<Argument>& schema_args,
+    at::ArrayRef<Value*> actual_inputs);
+
```

Comment:
I see that you moved this utility helper into `schema_matching` in order to make it accessible in multiple places. However, I don't seem the link between `calculateNecessayArgs` and `schema_matching`. Could you clarify? What do you think about putting it in a separate utility lib file?

Reply:
Sounds good. I just put there because they both reference Argument haha